frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -278,10 +278,13 @@
       </b-tabs>
       <b-pagination-nav
         v-model="currentPage"
-        size="lg"
+        base-url="#"
+        first-number
+        last-number
+        :number-of-pages="10"
         align="center"
+        size="lg"
         :link-gen="linkGen"
```

Comment:
can these items preserve the original order?

```suggestion
        base-url="#"
        first-number
        last-number
        size="lg"
        align="center"
        :link-gen="linkGen"
        :number-of-pages="10"
```

Reply:
Sure!